pkg/config/config.go

```diff
@@ -228,7 +269,8 @@ func (c SizeOrTimeLimitConfig) TimeLimitDuration(defaultVal time.Duration) time.
 	return val
 }
 
-func (c SizeOrTimeLimitConfig) timeLimitDuration() (val time.Duration, defaulted bool, err error) {
+// Deprecated: use PodLogLimits.SinceSeconds instead
```

Comment:
Just calling out the deprecation logic:

I think the way you did it here is fine; doing dual support for a time and then removing it altogether after another version or two.

Often, we move a little faster and are OK with breaking changes since we have a relatively short support window but I am not against this careful step since we are changing the config structure itself.

Can you open an issue stating that we need to remove these fields in v0.16? I think that is enough time to let it sit there.

Reply:
Done. https://github.com/heptio/sonobuoy/issues/789